swap_meet/vendor.py

```diff
@@ -1,2 +1,79 @@
+from swap_meet.item import Item
+
 class Vendor:
-    pass
+    
+    def __init__(self, inventory=[], items=[]):
```

Comment:
We need to be careful in Python if we use a mutable object for a default value. Consider:
```python
v1 = Vendor()
v2 = Vendor()
v1.add("item")
print(v2.inventory)  # this will print ['item']
```

There are a few ways to protect ourselves from this, depending on just how insulated from side effects we want to be:
1. Use an immutable default value (esp. `None`) and create a new mutable instance within the constructor, or
2. Always make a copy of the value being passed in

Reply:
I don't think there's any requirement that the `Vendor` constructor take an `items` list as a distinct parameter from the inventory. Inventory should end up storing the items that are passed in. Is `self.items` used anywhere else in this code?